src/Microsoft.DotNet.Wpf/src/WindowsBase/MS/Internal/Permissions/RightsManagementPermission.cs

```diff
@@ -27,25 +11,16 @@
 
 namespace MS.Internal.Permissions
 {
-    // !!!! Warning !!!!: No code other than DocumentApplication (or Mongoose) should assert this
-    //  permission without agreement from this code owners.
-    [Serializable]
     [FriendAccessAllowed]
     internal class RightsManagementPermission : InternalParameterlessPermissionBase
```

Comment:
This can also be easily deleted. The analysis is similar to that of `CompoundFileIOPermission`. 

`SecurityHelper.DemandRightsManagementPermission` is the only direct caller, which can be made into a no-op. 

In turn, `SecurityHelper.DemandRightsManagementPermission`  itself can be deleted, and its callers can be adjusted easily.